index.d.ts

```diff
@@ -26,6 +26,12 @@ declare module "webhook-discord" {
      */
     public warn(name: string, message: string): void;
 
+    /**
+     * This method is used to send a MessageBuilder to the webhook.
+     *
+     * @param {MessageBuilder} messageBuilder The message.
+     */
+    public send(messageBuilder: MessageBuilder): void;
```

Comment:
The return type here is not void, it's a promise. I'm not familiar with typescript myself but is this right?

```suggestion
    public send(messageBuilder: MessageBuilder): Promise;
```

Reply:
Oh, that's right. 
I just checked it and it should be a promise with an `undefined` return value.
In TS it would be `Promise<undefined>` then.